server/resources.py

```diff
@@ -44,10 +48,10 @@ def post(self):
 
 class AllUsers(Resource):
     def get(self):
-        return {'message': 'List of users'}
+        return UserModel.return_all()
 
     def delete(self):
-        return {'message': 'Delete all users'}
+        return UserModel.delete_all()
```

Comment:
I don't know if we want these routes right now and exposed? Maybe for testing, but we should remove them if we deploy / go to production

Reply:
Correct, it's only for testing purposes, and will be removed prior to going into production